### PR TITLE
Added 'MariaDB' in selection of running MySQL instances

### PR DESCRIPTION
**Pull request by kleinski**
Added 'MariaDB' in selection of running MySQL instances in mk_mysql.vbs.

### CODE REVIEW

**Comment by jherbel** [comment]
Hi, thank you for this PR. Just to be sure: Have you tried out that the rest of the script works with MariaDB?

**Comment by smokey42** [comment]
@kleinski I would add explicit parenthesis around the `or` section, because the order of precedence between `and` and `or` is not easily visible.

File: agents/windows/plugins/mk_mysql.vbs [hunk 1]
```diff
@@ -24,7 +24,7 @@ cfg_dir = SHO.ExpandEnvironmentStrings("%MK_CONFDIR%")
 '
 
 Set WMI = GetObject("winmgmts:{impersonationLevel=impersonate}!\\.\root\cimv2")
-Set service_list = WMI.ExecQuery("SELECT * FROM Win32_Service WHERE Name LIKE '%MySQL%' and State = 'Running'")
+Set service_list = WMI.ExecQuery("SELECT * FROM Win32_Service WHERE Name LIKE '%MySQL%' or Name LIKE '%MariaDB' and State = 'Running'")
```
Comment (smokey42) [inline agents/windows/plugins/mk_mysql.vbs:27]: You sould use something like `(Name LIKE '%MySQL%' or Name LIKE '%MariaDB')` to make it more obvious that both variants should be running.

**Comment by kleinski** [comment]
> Hi, thank you for this PR. Just to be sure: Have you tried out that the rest of the script works with MariaDB?

Yes, looks good.

Attached the output of the plugin. 
[mk_mysql-mariadb-output.txt](https://github.com/tribe29/checkmk/files/5920351/mk_mysql-mariadb-output.txt)

File: agents/windows/plugins/mk_mysql.vbs (hunk 1 above)
Comment (kleinski) [reply agents/windows/plugins/mk_mysql.vbs:27]: I do agree totally.

**Pull request closed without merging**